### PR TITLE
feat: Added Cart Page

### PR DESCRIPTION
**Pull request by vishal804**
Hello,

Develop Cart page for Ecommerce project. 

Please Review 

Thank You

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -50,4 +51,25 @@ main .page-container {
...
+/* Global Card Style */
+
+.card-style .card {
+  background-color: var(--background-color);
+  border: 1px solid var(--secondary-text-color);
+  margin: 0;
+}
+
+.card-image-container {
+  height: 250px;
```
Comment (Hamza-777) [inline css/styles.css]: Try to use rem or any other responsive units as much as possible.

**Review by Hamza-777 — APPROVED** [review]
Looks really beautiful..

File: css/styles.css (hunk 1 above)
Comment (vishal804) [reply css/styles.css]: Done the Changes 
Thank You for Review

**Review by jalajcodes — COMMENTED** [review]
Code looks good. Page is also looking nice, good work!

**Comment by vishal804** [comment]
> Code looks good. Page is also looking nice, good work!

Thanks For Review

**Pull request merged**